examples/flutter_gallery/lib/gallery/drawer.dart

```diff
@@ -162,7 +162,7 @@ class GalleryDrawer extends StatelessWidget {
               ]
             )
           ),
-          new DrawerItem(
+          onShowPerformanceOverlayChanged == null ? new Container(height: 0.0) : new DrawerItem(
```

Comment:
LGTM, but this is a little sketchy. We generally prefer to build the list of children and just not include those that we don't want rather than including dummy widgets.